### ISSUE TITLE
Butchery system does not scale down properly to accommodate smaller monsters

### ISSUE THREAD

**Issue by Regularitee** [CleverRaven/Cataclysm-DDA#24909]
The butchery system seems to neglect enemy sizes, or at least not scale properly.
1) Small enemies like rats, snakes, or baby spiders yield full-size corpses (30L, 50kg).
2) Overkill on enemies similarly yields excessive meat. (Blowing apart a baby spider yields a shower of 10 chunks of meat. A sewer snake yields 14 chunks of meat)

**Comment by nexusmrsep** [comment]
Problem # 2 is a direct result of problem # 1 - a SMALL sized creature, like a dog weights 40 kg:

`case MS_SMALL:  ret =  40750_gram;  break;`

As a result the balancing for meat yield was like this:

- 1 meat chunk is 200 g = 1/5 of 1 kg

'case MS_SMALL:  return 16;' and **it's already reduced to 1/4 of what it should be** (SIC!) for a 40 kg beast --> 64 chunks; 12800_gram (31% body mass)

Simplest way of balancing would be rigorous lowering body mass of a SMALL animal to perhaps 10 kg?
10 kg animal should yield 16-17 chunks = 3300_gram (33% body mass).
Secondly moving some small animals to tiny animals if necessary.
Might need other dev's opinion on that.

**Comment by jim-huynh** [comment]
What if we use the current lore as an excuse to reduce the effective yield closer to the original numbers?

If I remember correctly, everything is infected with blob.  So we could divide the yield by half, and say that half of the meat/fat you get becomes tainted meat/fat.

**Comment by Rivet-the-Zombie** [comment]
>What if we use the current lore as an excuse to reduce the effective yield closer to the original numbers?

>If I remember correctly, everything is infected with blob. So we could divide the yield by half, and say that half of the meat/fat you get becomes tainted meat/fat.

The infection isn't something you can see with the naked eye, and creatures aren't supposed to be half-rotten on the inside as a result of it either.

**Comment by nexusmrsep** [comment]
I have to agree with @Rivet-the-Zombie here. Problem here lies in size categories. I believe that yields should be bumped up even more as a direct derivative of body weight, like @ymber suggested in #24921 .

**Comment by jim-huynh** [comment]
Huh, I've always assumed that you could see the infection effects on meat because you get tainted meat from zombie bears rather than meat (rotten).

I also thought that it would work because the revivification (zombification?) starts after death, so a growing portion of the regular meat would become tainted meat given time.
